docs/metadata-ui.rst

```diff
@@ -0,0 +1,190 @@
+Import metadata using the Populate Metadata script in OMERO.web
+===============================================================
+
+Description
+-----------
+
+This chapter will show how to import metadata starting from a local CSV file and ending with OMERO.tables on images or wells using the `server-side script <https://omero-guides.readthedocs.io/en/latest/scripts/docs/index.html>`__ Populate Metadata in OMERO.web. See also the workflow described in :doc:`metadata` which will give you more possibilities, such as Key-Value Pairs creation, but which is using Command Line Interface (CLI). The workflow described here is using only graphical user interface elements.
+
+This action is typically done after a successful import of images.
+
+We will show:
+
+- How to import metadata from local CSV file in a bulk manner and turn them into OMERO.tables on
+   - :ref:`Images contained in Projects/Datasets<Populatemetadatapdi>`
+   - :ref:`Wells contained in Screens/Plates<Populatemetadataspw>`
+
+- How to :ref:`create or adjust a local CSV file containing metadata<Populatemetadatacsv>` for creation of OMERO.tables containing numbers and text. 
+
+Resources
+---------
+
+-  Data: example images from
+
+   -  IDR data `idr0021-lawo-pericentriolarmaterial/experimentA <https://idr.openmicroscopy.org/webclient/?show=project-51>`_
```

Comment:
Do we really need these images? Any images will do, right? (you ask the user to rename them anyway). If the user thinks they have to download IDR data, this could slow them down or block them.

Reply:
fixed in https://github.com/ome/omero-guide-upload/pull/13/commits/1620d675b85c298b9ce27985401500648ece2aed